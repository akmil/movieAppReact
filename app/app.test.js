import React from 'react';
// import ReactDOM from 'react-dom';
// import MainLayout from "./mainLayout";
import renderer from 'react-test-renderer';
import MainLayout from "./mainLayout";
import { shallow } from 'enzyme'

describe('MainLayout', function () {
    it('should render without errors', () => {
      // const wrapper = renderer.create(<MainLayout/>)
        const wrapper = shallow(<MainLayout/>);

        expect(wrapper).toMatchSnapshot();
    })

    /*it('should correctly calculate salary' , () => {
        const result = calculateSalary({ testsWritten: 5 });

        expect(result).toMatchSnapshot();

        expect(result).toEqual({
            prop1: {
                prop2: {
                    value: 5,
                }
            },
            prop3: {}
            prop8: {
                result: "No salary on this month"
            }
        })
    })*/
});
