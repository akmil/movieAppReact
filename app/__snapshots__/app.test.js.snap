// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout should render without errors 1`] = `
<div
  className="main-layout-wrapper"
>
  <Header1 />
  <ResultsBox />
  <FilmsBox />
</div>
`;
